composer.json

```diff
@@ -36,11 +36,11 @@
     ],
     "require-dev": {
         "doctrine/coding-standard": "^9.0.0",
-        "phpstan/phpstan": "0.12.98",
+        "phpstan/phpstan": "0.12.99",
         "phpunit/phpunit": "^9.5.9",
         "vimeo/psalm": "^4.10.0",
-        "roave/infection-static-analysis-plugin": "^1.9.0",
-        "infection/infection": "^0.24.0"
+        "roave/infection-static-analysis-plugin": "^1.10.0",
+        "infection/infection": "^0.25.1"
```

Comment:
I wonder if this dependency can be dropped overall, since we don't depend on any sources of `infection/infection`

Reply:
It can.

Reply:
```
Package operations: 1 install, 0 updates, 1 removal
  - Removing composer/package-versions-deprecated (1.11.99.4)
  - Installing ocramius/package-versions (2.3.0): 
```

I'm not sure if this will work...

Reply:
In theory, package-versions should work just fine with PHP 8.0, but I haven't bumped version ranges there yet :D 

I'm probably going through a lot of `ocramius/` packages tonight, if that's a blocker

Reply:
Yes, it failed https://github.com/Roave/BetterReflection/runs/3641721295?check_suite_focus=true :)

So I left the depency as it is and wait for green build (I hope) :)

Reply:
No worries - again, lack of dependabot for even just 1 month is really showing...